### PR TITLE
Adding improvements to sample scripts to support copying necessary files and cleaning bin.xml

### PR DESCRIPTION
**Pull request by thiliA**

### CODE REVIEW

File: modules/distribution/scripts/wso2cep-samples.sh [hunk 1]
```diff
@@ -66,7 +66,14 @@ do
       SAMPLE=""
       CARBON_HOME=`cd "$PRGDIR/.." ; pwd`
       CMD="$CMD -Daxis2.repo=$CARBON_HOME/samples/artifacts/$c"
-      `[ -f $CARBON_HOME/samples/artifacts/$c/stream-definitions.xml ] && cp $CARBON_HOME/samples/artifacts/$c/stream-definitions.xml $CARBON_HOME/repository/conf/data-bridge`
+      if [ ! -d $CARBON_HOME/samples/artifacts/$c/webapps ]; then 
+        `mkdir -p $CARBON_HOME/samples/artifacts/$c/webapps`
+        `cp -r $CARBON_HOME/repository/deployment/server/webapps/* $CARBON_HOME/samples/artifacts/$c/webapps/`
```
Comment (suhothayan) [inline modules/distribution/scripts/wso2cep-samples.sh]: We should not copy all the files but only the common ones

File: modules/distribution/scripts/wso2cep-samples.sh [hunk 2]
```diff
@@ -66,7 +66,14 @@ do
...
+      fi
+      if [ ! -d $CARBON_HOME/samples/artifacts/$c/jaggeryapps ]; then
+              `mkdir -p $CARBON_HOME/samples/artifacts/$c/jaggeryapps`
+              `cp -r $CARBON_HOME/repository/deployment/server/jaggeryapps/* $CARBON_HOME/samples/artifacts/$c/jaggeryapps/`
```
Comment (suhothayan) [inline modules/distribution/scripts/wso2cep-samples.sh]: We should not copy all the files but only the portal

File: modules/distribution/src/assembly/bin.xml [hunk 3]
```diff
@@ -118,13 +115,7 @@
                 <include>*.jar</include>
             </includes>
         </fileSet>
-        <fileSet>
-            <directory>../../modules/samples/target/output-ui-test</directory>
-            <outputDirectory>${pom.artifactId}-${pom.version}/samples/artifacts/0071/webapps</outputDirectory>
```
Comment (suhothayan) [inline modules/distribution/src/assembly/bin.xml]: This is only for test hence it has to be copied to the sample

File: modules/distribution/scripts/wso2cep-samples.sh (hunk 1 above)
Comment (thiliA) [inline modules/distribution/scripts/wso2cep-samples.sh]: Fixed in commit 2ad29af

File: modules/distribution/scripts/wso2cep-samples.sh (hunk 2 above)
Comment (thiliA) [inline modules/distribution/scripts/wso2cep-samples.sh]: Fixed in commit 2ad29af

File: modules/distribution/src/assembly/bin.xml (hunk 3 above)
Comment (thiliA) [inline modules/distribution/src/assembly/bin.xml]: Fixed in commit 2ad29af

**Pull request merged**